source/slang/hlsl.meta.slang

```diff
@@ -2341,10 +2346,11 @@ ${{{{
             return __texture_gather$(cmp)_offsets<T>(this,$(samplerStateParam) location $(compareArg), offset1,offset2,offset3,offset4 $(glslComponent));
         }
     }
-    ${{{{
+${{{{
+            } // for (isStatus)
```

Comment:
(not a suggestion)
I really like the idea of doing this to make it easier to read inline C++, I will be doing this from now on as well.